extensions/xcode/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "xcode",
+  "title": "Xcode",
+  "version": "1.0.0",
+  "author": "Sven Tiigi",
+  "license": "MIT",
+  "description": "Open your recent Xcode Projects, view the latest Xcode Releases and manage your Xcode Simulators",
+  "icon": "icon.png",
+  "commands": [
+    {
+      "name": "projects.command",
+      "title": "Projects",
+      "subtitle": "Your recent Xcode Projects",
```

Comment:
Here and in other commands I would just recommend writing "Xcode". Subtitle is usually 1-2 words adding some context. Most often it's the app / service it belongs to.

E.g. title "Create Issue" and subtitle "Jira".

Reply:
Fixed with https://github.com/raycast/extensions/pull/14/commits/9a8e1d863499426fd85a354069115e10626de53a